package.json

```diff
@@ -35,14 +35,14 @@
     "update:pkgs": "npx npm-check-updates -u"
   },
   "devDependencies": {
-    "autoprefixer": "^10.4.20",
+    "autoprefixer": "^10.4.21",
     "cross-env": "^7.0.3",
-    "hugo-extended": "0.145.0",
-    "postcss-cli": "^11.0.0",
+    "hugo-extended": "0.146.7",
```

Comment:
We're not ready to upgrade Hugo yet. For details, see:

- https://github.com/google/docsy/issues/2224

For example, see https://deploy-preview-355--goldydocs.netlify.app/docs/getting-started/example-page/#header-3, which demonstrates how codeblocks can get mangled:

> <img width="473" alt="image" src="https://github.com/user-attachments/assets/22078515-cb57-4682-9659-c7c95cb6ee44" />